sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SdkSpan.java

```diff
@@ -465,11 +472,13 @@ public ReadWriteSpan recordException(Throwable exception, Attributes additionalA
       additionalAttributes = Attributes.empty();
     }
 
+    int maxAttributeLength = spanLimits.getMaxAttributeValueLength();
     AttributesMap attributes =
         AttributesMap.create(
             spanLimits.getMaxNumberOfAttributes(), spanLimits.getMaxAttributeValueLength());
 
-    AttributeUtil.addExceptionAttributes(exception, attributes::put);
+    AttributeUtil.addExceptionAttributes(
```

Comment:
Do we know if the plan is to have `Span#recordException` eventually route the exception to the log SDK instead?  Or is the expectation that the instrumentation will migrate to a new API that uses the log SDK directly?

Reply:
> Do we know if the plan is to have `Span#recordException` eventually route the exception to the log SDK instead?

No, this would be considered a breaking change and could only possibly be done in an SDK 2.0 (which we don't currently have any plans for).

> Or is the expectation that the instrumentation will migrate to a new API that uses the log SDK directly?

Yeah, this is the plan currently at least. And instrumentation should probably take a major version bump when doing this migration to indicate the breaking nature of the change to its users.